packages/react-core/src/components/MultipleFileUpload/MultipleFileUploadButton.tsx

```diff
@@ -9,19 +9,21 @@ export interface MultipleFileUploadButtonProps extends React.HTMLProps<HTMLDivEl
   className?: string;
   /** Aria-label for the button */
   'aria-label'?: string;
+  'label'?: string;
 }
 
 export const MultipleFileUploadButton: React.FunctionComponent<MultipleFileUploadButtonProps> = ({
   className,
   'aria-label': ariaLabel,
+  label,
```

Comment:
```suggestion
  label = "Upload",
```

Right now the button is rendered without any text label